packages/core/test/specs/data_sources/model/StyleDataVariable.ts

```diff
@@ -198,12 +200,19 @@ describe('StyleDataVariable', () => {
         },
       ]);
 
+      cssComposer.render();
+      const view = rule.getView();
+
+      expect(rule.getStyle()).toHaveProperty('color', 'red');
       expect(em.getEditor().getCss()).toContain(`${selector}{color:red;}`);
+      expect(view?.el.innerHTML).toContain(`h1{color:red;}`);
 
       const ds = dsm.get(dsId);
       ds.getRecord(drId)?.set({ value: 'blue' });
 
+      expect(rule.getStyle()).toHaveProperty('color', 'blue');
       expect(em.getEditor().getCss()).toContain(`${selector}{color:blue;}`);
+      expect(view?.el.innerHTML).toContain(`h1{color:blue;}`);
```

Comment:
See now we have a `rule.getView()` method, that we assert updates to before and after data source update.